### PR TITLE
Added some sample code and PixelFormat constants

### PR DESCRIPTION
**Pull request by jpmikkers**

### CODE REVIEW

File: WIC/Constants/WICPixelFormats.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace WIC.Constants
+{
+    public static class WICPixelFormats
+    {
+        public static readonly Guid GUID_WICPixelFormatDontCare = new Guid(0x6fddc324, 0x4e03, 0x4bfe, 0xb1, 0x85, 0x3d, 0x77, 0x76, 0x8d, 0xc9, 0x00);
```
Comment (stakx) [inline WIC/Constants/WICPixelFormats.cs]: Could you please remove the `GUID_` name prefix from this and all following fields?

(I know that's what the documentation and original header files call it, but there's no added benefit in this prefix; I've taken some liberties with renaming things elsewhere to get a little closer to .NET's usual conventions.)

File: WIC/Constants/WICPixelFormats.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace WIC.Constants
+{
+    public static class WICPixelFormats
```
Comment (stakx) [inline WIC/Constants/WICPixelFormats.cs]: Consider renaming this class to `WICPixelFormat` (without `s`) for consistency with other preexisting enum-like typed in `Constants/`.

**Review by stakx — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! :+1: I'll be very happy to merge your contributions, just two small suggestions regarding code convention consistency.

**Comment by jpmikkers** [comment]
Done!

**Review by stakx — APPROVED** [review]

**Comment by stakx** [comment]
Wonderful, thank you!

Now that the source code is no longer just by me, I'll probably need to update the copyright in the license file.

Let me know if you'd like me to push an updated package to NuGet.

**Comment by jpmikkers** [comment]
Hi stakx, no that's not necessary, I barely touched the library.

**Pull request merged**